### PR TITLE
WIKI-1273 Move common utils methods to Commons Project

### PR DESCRIPTION
**Pull request by boubaker**

### CODE REVIEW

File: application/common/src/main/java/org/exoplatform/forum/common/user/CommonContact.java [hunk 1]
```diff
@@ -16,6 +16,7 @@
  */
 package org.exoplatform.forum.common.user;
 
+import org.exoplatform.commons.utils.CommonsUtils;
 import org.exoplatform.forum.common.CommonUtils;
```
Comment (thomasdelhomenie) [inline application/common/src/main/java/org/exoplatform/forum/common/user/CommonContact.java]: Import not used anymore.

File: application/common/src/main/java/org/exoplatform/forum/common/webui/BuildRendering.java [hunk 2]
```diff
@@ -26,6 +26,7 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
+import org.exoplatform.commons.utils.CommonsUtils;
 import org.exoplatform.forum.common.CommonUtils;
```
Comment (thomasdelhomenie) [inline application/common/src/main/java/org/exoplatform/forum/common/webui/BuildRendering.java]: Import not used anymore (to apply in all classes).

File: application/common/src/main/java/org/exoplatform/forum/common/user/CommonContact.java (hunk 1 above)
Comment (boubaker) [reply application/common/src/main/java/org/exoplatform/forum/common/user/CommonContact.java]: done

File: application/common/src/main/java/org/exoplatform/forum/common/webui/BuildRendering.java (hunk 2 above)
Comment (boubaker) [reply application/common/src/main/java/org/exoplatform/forum/common/webui/BuildRendering.java]: done

**Pull request merged**